int8 spi_cmd
